src/char/barbarian.py

```diff
@@ -1,4 +1,5 @@
 import keyboard
+from ui_components.skills import is_left_skill_selected, is_right_skill_active
```

Comment:
```suggestion
import ui_compoents.skills as skills
```

Reply:
This will allow a usage pattern like
```python
skills.is_left_skill_selected()
```
Much better discoverability of other things you put there over time

Reply:
Can we apply this to all other usages too?

Reply:
Fixed